coala_quickstart/green_mode/bear_settings.yaml

```diff
@@ -0,0 +1,94 @@
+type4:
+  CMakeLintBear:
+    cmakelint_config: ''
+  JSHintBear:
+    jshint_config: ''
+
+type3:
+  FilenameBear:
+    file_naming_convention: ['auto', 'camel', 'kebab', 'pascal',
+                             'snake', 'space']
+    # NOTE: Setting it to auto guesses the correct convention
+    #       by the bear itself. Defaults to snake if the correct
+    #       convention cannot be guessed.
+  GNUIndentBear:
+  # discarding `indent_cli_options` as all options already handled by
+  # other bear settings.
+  CPDBear:
+    language: ['C#', 'C++'] # As currently IMPORTANT_BEAR_LIST
+    # and GREEN_MODE_COMPATIBLE_BEAR_LIST use it for only these 2 languages.
+  CPPCheckBear:
+    enable: ['all', 'warning', 'style', 'performance',
+             'portability', 'information', 'unusedFunction',
+             'missingInclude']
+  PycodestyleBear:
+    pycodestyle_ignore: ['D100', 'D101', 'D102', 'D103', 'D104',
+                        'D105','D106', 'D107',
+                        'D200', 'D201', 'D202', 'D203', 'D204',
+                        'D205', 'D206', 'D207', 'D208', 'D209',
+                        'D210', 'D211', 'D212', 'D213', 'D214',
+                        'D215',
+                        'D300', 'D301', 'D302',
+                        'D400', 'D401', 'D402', 'D403', 'D404',
+                        'D405', 'D406', 'D407', 'D408', 'D409',
+                        'D410', 'D411', 'D412', 'D413', 'D414']
+    pycodestyle_select:  ['D100', 'D101', 'D102', 'D103', 'D104',
+                          'D105','D106', 'D107',
+                          'D200', 'D201', 'D202', 'D203', 'D204',
+                          'D205', 'D206', 'D207', 'D208', 'D209',
+                          'D210', 'D211', 'D212', 'D213', 'D214',
+                          'D215',
+                          'D300', 'D301', 'D302',
+                          'D400', 'D401', 'D402', 'D403', 'D404',
+                          'D405', 'D406', 'D407', 'D408', 'D409',
+                          'D410', 'D411', 'D412', 'D413', 'D414']
+  JSHintBear:
+    es_version: [3, 5, 6]
+    allow_latedef: [True, False, 'nofunc']
+    allow_unused_variables: [True, False, 'vars', 'strict']
+    allow_variable_shadowing: [True, False, 'inner', 'outer']
+    javascript_strictness: [True, False, 'implied', global]
+
+  # The following settings are Type 2 but migrated to Type 3 because
+  # of probable values and lack of efficient algorithms to deduce their
+  # appropriate values from the prohect.
```

Comment:
typo

Reply:
I'm wondering if we were to generate this .yml on the fly how would we handle such cases (those you mentioned in the comments throughout the file)